libsolidity/lsp/FileRepository.cpp

```diff
@@ -51,6 +67,8 @@ string FileRepository::sourceUnitNameToUri(string const& _sourceUnitName) const
 		return "file:///" + _sourceUnitName;
 	else if (_sourceUnitName.find("/") == 0)
 		return "file://" + _sourceUnitName;
```

Comment:
Doesn't this break resolution for files that are placed directly under one of the include dirs? I.e. if you have LSP set up equivalent to `--base-path /a/ --include-path b/c` then you won't be able to do `import "x.sol"` to import `/a/b/c/x.sol`, it will look for `/a/x.sol` instead.

Reply:
@cameel: If at all, I think this if-stmt should be removed all together. Meaning, it should not happen (AFAICT). I might be wrong. But if at all, I'd prefer to remove this in a seperate PR, because these lines are unrelated to the actual PR that should not become too big. What do you think?

Reply:
Fair enough. If it was there already I think it's fine to address it separately.

Reply:
@cameel I've added a test (bad-include) and it triggers a diagnostic that the given file "test.sol" could not be found.

Reply:
ok, so you'll be fixing that in a separate PR?

Reply:
I see this has been removed to marking as resolved.